### ISSUE TITLE
Platform error sensor.paxcalima - Integration 'paxcalima' not found.

### ISSUE THREAD

**Issue by timbba80** [MarkoMarjamaa/homeassistant-paxcalima#1]
I'm trying to install this to Home Assistant running in raspberry 4. 

Issue: 

When going to server controls ->check configuration, I'm getting an error message "Platform error sensor.paxcalima - Integration 'paxcalima' not found.".

Steps done: 

1. I've copied _init_.py, manifest.json and sensor.py to /config/custom_components/paxcalima
2. Reboot home assistant
3. Include following to configuration.yaml under "sensors" (I have bunch of other platforms under sensors, which do work fine)
  - platform: paxcalima
    mac: xx:xx:xx:xx:xx:xx
    pin: xxxxxxxx
4. Error message given for integration not found. 

I do have bunch of other custom components installed, and they are working fine. 

I'm running home assistant OS core-2021.9.7 in Raspb 4.

**Comment by MarkoMarjamaa** [comment]
This works with HA version 0.108.9. 
HA likes to change integration interface often so that's probably the cause. 
Every time they change the integration interface, every integration should also be updated. 
With custom integrations it is difficult. That's why because I don't update often, because I have also other custom integrations. 
Blame HA...

**Comment by timbba80** [comment]
This can be closed, I noticed that @petrepa has created a fork, which contains updated manifest.json. 

Second requirement was that calima is made available in pypi. I've now created calima package to pypi, which also takes care of the installation (without requirement to proceed with the manual steps). 

So with these it should work by only copying sensor.py, manifest.json and _init_.py to custom_components folder, rebooting and then modifying configuration.yaml.
